src/modules/filelist/icons/FileThumbnail.jsx

```diff
@@ -95,4 +100,9 @@ FileThumbnail.propTypes = {
   size: PropTypes.number
 }
 
+FileThumbnail.defaultProps = {
+  showSharedBadge: false,
+  componentsProps: {}
```

Comment:
```suggestion
    componentsProps: {
        sharedBadge: {}
    }
```
Ce qui permettrai d'éviter la condition `componentsProps.sharedBadge ?? {}` plus haut ?